### ISSUE TITLE
Accessibility - Count down

### ISSUE THREAD

**Issue by goranmiric** [ractoon/jQuery-Text-Counter#48]
When typing screen reader doesn't announce the number of character remaining, and that represents an accessibility issue.

**Comment by goranmiric** [comment]
Here is the PR for it: https://github.com/ractoon/jQuery-Text-Counter/pull/49

**Comment by ractoon** [comment]
Merged. Thanks!
